File: Basics_code_igenuine/java/works/fact.java
import java.util.Scanner;
public class fact
{
	public static void main(String[]args)
	{
		int n,i;
		long fact=1;
		Scanner sc=new Scanner(System.in);
		System.out.println("enter the number");
		n=sc.nextInt();
		sc.close();
		if(n<=0)
		{
			System.out.println("Error");
		}
		else
		{
			for(i=1;i<=n;i++)
			{
				fact*=i;
			}
		}
	System.out.println("The factorial of "+n+" is "+fact);	
	}
}

File: Basics_code_igenuine/php/myweb/works/fact.php
<?php
function factorial($a)
{ $fact=1;
	if($a<0)
	{
		echo "Error,Factorial of negative number doesn't exist....";
		return -1;
	}
		else
		{
			for($i=1;$i<=$a;++$i)
			{
				$fact*=$i;
			}
			return $fact;
		}
		
}
?>
<html>
<head>
</head>
<body>
<h2>Factorial</h2>
<?php
if(!isset($_POST['submit']))
{
?>
	<form  id="btn" method="post" action="fact.php">
	Enter the number:<br/><br/>
	<input type="text" name="num_1" placeholder="Enter the number" size="20"/>

	<input type="submit" name="submit"  value="Submit"/>
	</form>
<?php	
}
else
{
	$num=(int)$_POST['num_1'];
	echo "The factorial of the number $num is ".factorial($num);
	
}
?>
</body>
</html>